client/src/components/SymptomsTracker.js

```diff
@@ -4,6 +4,7 @@ import { Typography, Slider, ButtonGroup, Button, Checkbox, FormControlLabel, Te
 
 // custome style for material ui elements
 const useStyles = makeStyles({
+<<<<<<< HEAD
```

Comment:
@crispinamuriel These tags indicate merge conflicts and need to be removed.
See https://stackoverflow.com/questions/7901864/git-conflict-markers for more explanations.